ledger/streaming/client/src/scp_validator.rs

```diff
@@ -180,17 +182,18 @@ impl<ID: GenericNodeId + Send + Clone> SCPValidationState<ID> {
 
         // Determine if the blocks we've collected so far are a quorum slice
         if let Some(tracked_blocks) = self.slots_to_externalized_blocks.get(&index) {
-            for member in quorum_set.members.iter() {
+            for member in &quorum_set.members {
+                let member = member.as_ref();
                 // Go through our slice and determine what nodes sent blocks AND
                 // what block they externalized
                 match member {
-                    QuorumSetMember::Node(node_id) => {
+                    Some(QuorumSetMember::Node(node_id)) => {
                         if let Some(block_data) = tracked_blocks.get(node_id) {
                             // Record a vote for the block externalized
                             ballot_map
                                 .entry(&block_data.block().id)
-                                .and_modify(|vec| vec.push(member))
-                                .or_insert_with(|| vec![member]);
+                                .or_default()
+                                .push(member.unwrap());
```

Comment:
While I understand that `member.unwrap()` shouldn't fail here and there are already some `unwrap()` calls in this module, it still feels off to have an `unwrap()`.
Perhaps something like this:
```suggestion
                    Some(node @ QuorumSetMember::Node(node_id)) => {
                        if let Some(block_data) = tracked_blocks.get(node_id) {
                            // Record a vote for the block externalized
                            ballot_map
                                .entry(&block_data.block().id)
                                .or_default()
                                .push(node);
```